VideoLocker/AndroidManifest.xml

```diff
@@ -93,28 +93,27 @@
 
         <activity
             android:name=".view.CourseDiscussionTopicsActivity"
+            android:configChanges="orientation|keyboardHidden|screenSize"
             android:label="@string/app_name"
-            android:screenOrientation="portrait"
-            >
-        </activity>
+            android:screenOrientation="user" />
```

Comment:
We could just leave this attribute undefined, and thus resolved to the system default setting.